contents/stats_distributions.md

```diff
@@ -0,0 +1,159 @@
+In other words, distributions are a mapping between values and their respective probabilities.
+We generally denote that a random variable $X$ follows a specific distribution by the following notation:
+
+$$ X \sim \operatorname{dist}(\theta_1, \theta_2, \dots), $$ {#eq:distribution_notation}
+
+where $\operatorname{dist}$ is the name of the distribution and $\theta_1, \theta_2, \dots$ are the parameters that control how the distribution behaves.
+Every distribution can be "parameterized" by specifying parameters that allow us to tailor some aspects of the distribution for some specific purpose.
+
+Also, notice that we are using upper case $X$ instead of lower case $x$.
+This is a wide used convention in probability and statistics, that **upper case characters denote random variables, and lower case characters denote numerical values of a random variable**.
+More broadly, $X$ is a theoretical distribution, whereas $x$ are the observed values from $X$.
+
+There are several distributions defined in textbooks[^stats_book] and scientific articles[^stats_articles].
+We'll cover just a few of them in this section.
+In Julia, we have a package that provides a large collection of distributions and related function called [`Distributions.jl`](https://juliastats.org/Distributions.jl/dev/), which we will use to showcase some distributions:
+
+[^stats_book]: we recommend @bertsekas2000introduction.
+[^stats_articles]: one example is the $\operatorname{LKJ}$ distribution [@lewandowskiGeneratingRandomCorrelation2009].
+
+```julia
+using Distributions
+```
+
+### Normal and Non-Normal Distributions {#sec:stats_dist_normal}
+
+In applied sciences, there is a widely used distribution called the **normal distribution**.
+It has the ubiquitous shape of a bell curve and is also known as the *Gaussian* distribution in honor of the mathematician Carl Friedrich Gauss.
+
+This distribution is generally used in the social and natural sciences to represent variables in which their distributions are not known.
+Some phenomena follow a normal distribution, such as human height, human shoe size or test scores.
+
+We urge the reader to perceive that *not all phenomena are normally distributed*.
+For example, income, financial returns, city size, social media followers are examples of important phenomena that, generally, do not follow a normal distribution.
+Instead, they follow what we will call as non-normal distribution, or, as some also denote, a long-tailed distribution.
+
+In @fig:plot_normal_lognormal, we can see the comparison between a distribution that follows a normal distribution and a distribution that is long-tailed, i.e. does *not* follow a normal distribution.
+
+```jl
+fig = plot_normal_lognormal()
+caption = "Normal and Non-Normal Distributed Distributions."
+label = "plot_normal_lognormal"
+Options(fig; filename=label, caption, label)
```

Comment:
FYI, since yesterday, you can also show the code here without using `# hide` thanks to https://github.com/rikhuijzer/Books.jl/pull/209.

Then, it would be
```
let
    caption = "Normal and Non-Normal Distributed Distributions."
    label = "plot_normal_lognormal"
    pre(fig) = Options(fig; filename=label, caption, label)
    @sco pre=pre plot_normal_lognormal()
end
```
cc @lazarusA